script/build_docs.sh

```diff
@@ -29,7 +30,12 @@ do
   mkdir -p "$dest_dir"
   params="--chunk=1"
   if [ "$PREVIEW" = "1" ]; then
-    params="--chunk=1 -open chunk=1 -open"
+    params+=" -open chunk=1 -open"
```

Comment:
Seems to work without it as well. And the `chunk=1` also seems superfluous since right above it already sets `--chunk=1`. It works without both of them for me, so I'm removing both.